### ISSUE TITLE
About the tricks during the training

### ISSUE THREAD

**Issue by 2686873626** [yoctta/multiple-attention#17]
After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%.  Is there any other trick used during the training of attention layer.

**Comment by yoctta** [comment]
Then you should finetune the whole network with AGDA for some epochs.


From: 2686873626
Sent: Tuesday, February 22, 2022 12:11 PM
To: yoctta/multiple-attention
Cc: Subscribed
Subject: [yoctta/multiple-attention] About the tricks during the training (Issue #17)

After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%. Is there any other trick used during the training of attention layer.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by 2686873626** [comment]
> Then you should finetune the whole network with AGDA for some epochs. From: 2686873626 Sent: Tuesday, February 22, 2022 12:11 PM To: yoctta/multiple-attention Cc: Subscribed Subject: [yoctta/multiple-attention] About the tricks during the training (Issue #17) After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%. Is there any other trick used during the training of attention layer. — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Thank you very much

**Comment by damengdameng** [comment]
> > Then you should finetune the whole network with AGDA for some epochs. From: 2686873626 Sent: Tuesday, February 22, 2022 12:11 PM To: yoctta/multiple-attention Cc: Subscribed Subject: [yoctta/multiple-attention] About the tricks during the training (Issue #17) After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%. Is there any other trick used during the training of attention layer. — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> 
> Thank you very much

hi, can you share the environment details?  Thank you!

**Comment by 2686873626** [comment]
> > > Then you should finetune the whole network with AGDA for some epochs. From: 2686873626 Sent: Tuesday, February 22, 2022 12:11 PM To: yoctta/multiple-attention Cc: Subscribed Subject: [yoctta/multiple-attention] About the tricks during the training (Issue #17) After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%. Is there any other trick used during the training of attention layer. — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> > 
> > 
> > Thank you very much
> 
> hi, can you share the environment details? Thank you!

I've already sent it to your email

**Comment by xuyingzhongguo** [comment]
> > > > Then you should finetune the whole network with AGDA for some epochs. From: 2686873626 Sent: Tuesday, February 22, 2022 12:11 PM To: yoctta/multiple-attention Cc: Subscribed Subject: [yoctta/multiple-attention] About the tricks during the training (Issue #17) After the pretraining of backbone, I froze it and then train the multi-attentional without AGDA loss. But the accuracy is still no more than 97%. Is there any other trick used during the training of attention layer. — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> > > 
> > > 
> > > Thank you very much
> > 
> > 
> > hi, can you share the environment details? Thank you!
> 
> I've already sent it to your email

Hi, do you mind sharing it with me please? Thank you!
